osu.Game.Rulesets.Osu/Mods/OsuModTarget.cs

```diff
@@ -139,7 +139,7 @@ protected override void ApplyNormalVisibilityState(DrawableHitObject drawable, A
 
                 var avgColour = colour.AverageColour.Linear;
                 drawable.FadeColour(new Color4(avgColour.R * 0.45f, avgColour.G * 0.45f, avgColour.B * 0.45f, avgColour.A))
-                        .Then().Delay(h.TimePreempt - controlPointInfo.TimingPointAt(h.StartTime).BeatLength).FadeColour(colour);
+                        .Then().Delay(h.TimePreempt - controlPointInfo.TimingPointAt(h.StartTime).BeatLength - 96).FadeColour(colour, 96);
```

Comment:
constants like this need to be extracted out to `private const` fields, with xmldoc where the name isn't already explaining what they mean.